packages/console/Command/ExportListCommandsCommand.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Draw\Component\Console\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Process\Process;
+
+#[AsCommand(name: 'app:list-commands')]
+class ExportListCommandsCommand extends Command
+{
+    protected function configure(): void
+    {
+        $this
+            ->setDescription('List all available Symfony commands with their documentation.')
+            ->addOption('output', 'o', InputOption::VALUE_REQUIRED, 'Output file', 'commands_help.json');
```

Comment:
Ouptut should be a argument required not a option

Switch it back to the txt directly. No need to output in a json to after that do a template to generate the same output as the txt you had before.

Also the path can be in a folder that doesn't exist you chould create it. Use Symfony\Component\Filesystem\Filesystem to do so.

Reply:
I can add to arguments one for the `path` and one for the `filename`, or  is it overkill?